src/client/pages/AssessmentSection/DataTable/Table/Row/RowData/Cell/hooks/useOnChange.ts

```diff
@@ -38,12 +38,15 @@ export default (props: Props): UseOnChange => {
   const assessment = useAssessment()
   const assessmentSection = useAssessmentSection(sectionName)
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { estimated, estimationUuid, ...nodeValue } = _nodeValue ?? ({} as NodeValue)
```

Comment:
why this estlint disable? I think it's unnecessary. what about using the ?. to access nullable properties ?